.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - dev
+    pull_request:
+      branches: 
+        - '**'
+
+jobs:
+  build:
+
+    env:
+      APP_ENV: test
+    
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
```

Comment:
If we're only testing on one OS, it doesn't need to be in the matrix. I don't know if we want to add other OSs; it doesn't seem very useful for this project, although making sure it'll run on Mac and Windows might be good for developers.

Reply:
Got it!

Reply:
Does this apply for php as well? Do we need different versions?

Reply:
Wikimedia OCR explicitly only works on PHP 7.3 right now. If we move to VPS we'll probably bump to 7.4, so I suppose it doesn't hurt to keep testing against both 7.3 and 7.4

Reply:
Sounds good. Thank you!